module_utils/network_lsr/argument_validator.py

```diff
@@ -1569,3 +1602,12 @@ def validate_connection_one(self, mode, connections, idx):
                     "Configure wireless connection in /etc/wpa_supplicant.conf "
                     "if you need to use initscripts.",
                 )
+
+        # initscripts does not support ipv6_disabled, so raise errors when network
+        # provider is initscripts
+        if connection["ip"]["ipv6_disabled"]:
+            if mode == self.VALIDATE_ONE_MODE_INITSCRIPTS:
+                raise ValidationError.from_connection(
+                    idx,
+                    "ipv6.method disabled is not supported by initscripts.",
+                )
```

Comment:
@thom311 , I added the error handler for initscripts

Reply:
looks good. But the error message doesn't seem very suitable. "ipv6.method disabled" is something of NetworkManager. The paramter in the role is called "ip.ipv6_disabled" and the error message should refer to that.

Reply:
already changed into `"ip.ipv6_disabled is not supported by initscripts."`